sites/all/themes/orwell/templates/ting/ting-object--full.tpl.php

```diff
@@ -81,7 +81,7 @@
       <?php endif; ?>
 
       <?php // Render abstract if not already rendered above ?>
-      <?php if (!empty($content['group_ting_object_right_column']['ting_relations']['#groups']['dbcaddi:hasDescriptionFromPublisher'])) : ?>
+      <?php if (!empty($content['group_ting_object_right_column']['group_ting_object_right_c']['ting_abstract'])) : ?>
```

Comment:
The condition should negate the one used in line 42 to no render the abstract twice (it may be rendered in line 51 if `$content['group_ting_object_right_column']['ting_relations']['#groups']['dbcaddi:hasDescriptionFromPublisher']` is empty). This template is a mess …

The real issue must be that “dbcaddi:hasDescriptionFromPublisher” is suddenly not set.